CMakeLists.txt

```diff
@@ -43,12 +45,12 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
 #----------------------------------------------------------------------
-# Find python3 for the build system
+# Set up virtual environment for the build system
```

Comment:
note: Since we use the Python package `schema` to do validation, we need set up a virtual environment to do the build in that has that package.